doc/release/2.8.1.rst

```diff
@@ -0,0 +1,347 @@
+
+-   Fixed a crash which occurred when reloading a compiled module when the new module
+    lacked some of the functions which were present in the former code.
+    This event triggered a fallback procedure where Tarantool restored old
+    functions, but instead of restoring each function it only processed a sole
+    entry, leading to a crash later when these restored functions were called
+    (:tarantool-issue:`5968`).
+
+-   Added memtx MVCC tracking of read gaps which fixes the problem of phantom reads
+    (:tarantool-issue:`5628`).
+
+-   Fixed the wrong result of using ``space:count()`` with memtx MVCC (:tarantool-issue:`5972`).
+
+-   Fixed the dirty read after restart while using MVCC with synchronous
+    replication (:tarantool-issue:`5973`).
+
+
+Replication
+~~~~~~~~~~~
+
+-   Fixed an issue with an applier hanging on a replica after failing to process
+    a ``CONFIRM`` or ``ROLLBACK`` message coming from a master.
+
+-   Fixed the issue where master did not send some rows to an anonymous replica
+    which had fallen behind and was trying to register.
+
+-   Fixed the bug when a synchronous transaction could be confirmed and
+    visible on a replica, but then not confirmed or invisible again after
+    restart. It was more likely to happen on memtx spaces with
+    ``memtx_use_mvcc_engine`` enabled (:tarantool-issue:`5213`).
+
+-   Fixed the recovery of a rolled back multi-statement synchronous transaction
+    which could lead to the transaction being applied partially, and to
+    recovery errors. It happened in case the transaction worked with
+    non-sync spaces (:tarantool-issue:`5874`).
+
+-   Fixed a bug in synchronous replication when rolled back transactions
+    could reappear after reconnecting a sufficiently old instance
+    (:tarantool-issue:`5445`).
+
+
+Module swim
+~~~~~~~~~~~
+
+-   Fixed an issue where ``<swim_instance>:broadcast()`` did not work on non-local
+    addresses and spammed "Permission denied" errors to the log.
+    After instance termination it could return a non-0 exit code even if there
+    were no errors in the script, and then spam the same error again
+    (:tarantool-issue:`5864`).
+
+-   Fixed the crash on attempts to call ``swim:member_by_uuid()`` with no
+    arguments or with ``nil``/``box.NULL`` (:tarantool-issue:`5951`).
+
+-   Fixed the crash on attempts to pass an object of a wrong type to
+    ``__serialize`` method of a swim member in Lua (:tarantool-issue:`5952`).
+
+
+LuaJIT
+~~~~~~
+
+-   Lua stack resizing no longer results in a wrong behaviour of the memory profiler
+    (:tarantool-issue:`5842`).
+
+-   Fixed a double ``gc_cdatanum`` decrementing in LuaJIT platform metrics
+    which occurred when a finalizer was set for a ``GCсdata`` object (:tarantool-issue:`5820`).
+
+
+Lua
+~~~
+
+-   Fixed the `-e` option, when ``tarantool`` used to enter the interactive mode when
+    stdin is a TTY. Now, ``tarantool -e 'print"Hello"'`` doesn't enter the
```

Comment:
I would use the usual syntax with parenthesis (`tarantool -e "print('Hello')"`).

Reply:
what's the meaning of parentheses?

Reply:
A function call. I don't know why print works without parentheses. It seems, it is some weird exception from usual syntax rules.